### ISSUE TITLE
[a11y]: "Thinking..." system status must be announced

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#374]
### Browser

Firefox

### Operating System

Windows

### Package version

0.14.1-canary.347

### React version

^1.66.0

### Automated testing tool and ruleset

none

### Assistive technology

JAWS 2025

### Description

When a user submits a question to the chat assistant and the user is waiting on the system to return an answer, they cannot hear the "Thinking..." status shown visually on the screen so it is unclear to a non-sighted user what the current status is.

According to [4.1.3 status messages](https://www.ibm.com/able/requirements/requirements/#4_1_3), this status should be announced to users using screen readers.

### Reproduction/example

none

### Steps to reproduce

1. leverage the ai-chat component
2. enter a chat message
3. wait for a response

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
